benchmarks/benchmarks/bench_function_base.py

```diff
@@ -5,6 +5,34 @@
 import numpy as np
 
 
+class Histogram1D(Benchmark):
+    def setup(self):
+        self.d = np.linspace(0, 100, 100000)
+
+    def time_full_coverage(self):
+        np.histogram(self.d, 200, (0, 100))
+
+    def time_small_coverage(self):
+        np.histogram(self.d, 200, (50, 51))
+
+    def time_fine_binning(self):
+        np.histogram(self.d, 10000, (0, 100))
+
+
+class Histogram2D(Benchmark):
+    def setup(self):
+        self.d = np.linspace(0, 100, 200000).reshape((-1,2))
+
+    def time_full_coverage(self):
+        np.histogramdd(self.d, (200, 200), ((0, 100), (0, 100)))
+
+    def time_small_coverage(self):
+        np.histogramdd(self.d, (200, 200), ((50, 51), (50, 51)))
+
+    def time_fine_binning(self):
+        np.histogramdd(self.d, (10000, 10000), ((0, 100), (0, 100)))
```

Comment:
In the interest of seeing a comparison at https://pv.github.io/numpy-bench/, can you make a separate PR with just the benchmarks, that we can merge a day earlier?